_data/navi.yml

```diff
@@ -28,13 +28,26 @@ toc:
           url: constraints.html
         - title: Attributes
           url: attributes.html
-        - title: JSON Beispiel
-          url: json.html
         - title: XML Beispiel
           url: xmlbeispiel.html
         - title: XSD
           url: xsdschema.html
-
+  - title: Beispiele
+    path: /examples/
+    url: examples.html
+    subnav:
+      - title: Reine Metadatenanzeige
+        url: metadataonly.html
+      - title: Authentifizierung nötig
+        url: authentification.html
+      - title: Abschluss eines Nutzungsvertrags
+        url: agreement.html
+      - title: IP-Adressbereichs
+        url: location.html
+      - title: Ansicht, aber keine Speicherungs-/Druckmöglichkeit
+        url: readonly.html
+      - title: Beschränkung gleichzeitiger Zugriff
+        url: concurrent.html
```

Comment:
@Duchatelet Ich hab versucht die Beispiele in der Seitennavigation griffiger zu formulieren. Passt das aus deiner Sicht?

Reply:
Scheint mir besser :) 
Allerdings würde ich 2 bis 3 noch etwas korrigieren. 

- "IP-Adressbereichs" da sollte denke ich das S weg? 
- Der Titel "Ansicht, aber keine Speicherungs-/Druckmöglichkeit" kommt mir etwas indirekt vor. Also ich würde es bei "Reine Ansicht" belassen oder sowas ähnliches. Sonst müsste ja alles aufgelistet werden was man nicht darf.
- "Beschränkung gleichzeitiger Zugriff" fehlt da ein E am Ende oder stört das nur mich?

Reply:
> "IP-Adressbereichs" da sollte denke ich das S weg?

Stimmt. Da hatte ich es Einschränkung eines Bereichs oder sowas…

> "Reine Ansicht"

Reine Präsentation?

> "Beschränkung gleichzeitiger Zugriff" fehlt da ein E am Ende oder stört das nur mich?

Hmmmmmmm… das geht beides. Ich nehm das e dazu.